recipes/libnegf/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -ex
+
+if [ "${mpi}" != "nompi" ]; then
+  MPI=ON
+else
+  MPI=OFF
+fi
+
+cmake_options=(
+   "-DCMAKE_INSTALL_PREFIX=${PREFIX}"
+   "-DCMAKE_INSTALL_LIBDIR=lib"
+   "-DBUILD_SHARED_LIBS=ON"
+   "-DWITH_MPI=${MPI}"
+   "-GNinja"
+   ".."
+)
+
+mkdir -p _build
+pushd _build
+cmake "${cmake_options[@]}"
+
+ninja all install
```

Comment:
Is there a make check/test or ctest that we can run here?

Reply:
The only testsuite for libnegf I'm aware of is available in a dependent project.